packages/theme/src/cli/utilities/theme-environment/types.ts

```diff
@@ -23,9 +23,18 @@ export interface DevServerSession extends AdminSession {
   storefrontPassword?: string
 
   /**
-   * Timestamp marking when this session expires.
+   * The cookies that impact the rendering of the development server.
+   * Currently, we have two cookies that impact that:
+   *
+   *   - storefront_digest:  This cookie identifies an authenticated
+   *                         session and allows rendering to happen without
+   *                         the need for a password.
+   *
+   *   - _shopify_essential: This cookie essentially identifies the session,
+   *                         which is crucial to determine the theme being used
+   *                         during the rendering.
    */
-  expiresAt: Date
+  sessionCookies?: {[key: string]: string}
```

Comment:
Is this ever going to be actually undefined? Are there any side effects (aside from testing) of making this required?

https://github.com/Shopify/cli/blob/03e53f9d4fd929b346c61e96a5f76da04e6ff06d/packages/theme/src/cli/utilities/theme-environment/storefront-session.ts#L44

Reply:
Great call! `cookieRecord` is no longer nullable :)